examples/cloud-cognitive/ccs-base-react-17/src/App.test.js

```diff
@@ -3,6 +3,5 @@ import App from './App';
 
 test('renders learn react link', () => {
   render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+  screen.getByText(/Show AboutModal/i);
```

Comment:
A bit of a nitpick, but I think the name of this test should also be changed. Also, it's our normal convention to go for `it` rather than `test`.

Reply:
@lee-chase would be good to get this done and merged if there isn't a particular reason to hold it

Reply:
Done @dcwarwick